client/mycart/src/app/shared/navbar/navbar.component.html

```diff
@@ -26,7 +26,7 @@
                 </li>
             </ul>
             <div>
-                <a class="btn btn-info me-2 text-light login-btn" [routerLink]="['/customer/customerHome']"
+                <a class="btn btn-info me-2 text-light login-btn" [routerLink]="['/login']"
                     routerLinkActive="router-link-active">
                     Login</a>
                 <a class="btn btn-warning me-2 text-light register-btn">Register</a>
```

Comment:
Where is the href attribute or routerLine?

Reply:
Gave the router Line ad/login instead of /customer/customerHome